cmd/cert-checker/main.go

```diff
@@ -170,10 +175,11 @@ func (c *certChecker) checkCert(cert core.Certificate) (problems []string) {
 		if parsedCert.IsCA {
 			problems = append(problems, "Certificate can sign other certificates")
 		}
-		// Check the cert has the correct validity period +/- an hour
-		if parsedCert.NotAfter.Sub(cert.Issued) > checkPeriod+time.Hour {
+		// Check the cert has the correct validity period
+		expiryPeriod := parsedCert.NotAfter.Sub(parsedCert.NotBefore)
+		if expiryPeriod > checkPeriod {
 			problems = append(problems, "Certificate has a validity period longer than 90 days")
-		} else if parsedCert.NotAfter.Sub(cert.Issued) < checkPeriod-time.Hour {
+		} else if expiryPeriod < checkPeriod {
```

Comment:
I'm somewhat confuse about what your question is here? The point of the two checks here are to check that the certificate has the correct validity period (`NotAfter-NotBefore`). Technically we could just do a `!expiryPeriod.Equal(checkPeriod` but since this is for the auditors I thought it would be best to be as verbose as possible.